packages/schema/src/workspace/workspace.ts

```diff
@@ -0,0 +1,155 @@
+import { z } from 'zod'
+import { PageRequestSchema, PageResponseSchema } from '../index'
+
+export const CreateWorkspaceSchema = z.object({
```

Comment:
I'm not sure why you have added `CreateWorkspaceSchema` and `UpdateWorkspaceSchema` in here. We already have that below

Reply:
I kept it since the types were different in `api-client` and `schema`
https://github.com/keyshade-xyz/keyshade/blob/59e428002b3b9bb88908da6ffa81605de7d2b3dd/packages/api-client/src/types/workspace.types.d.ts#L16C1-L19C2

https://github.com/keyshade-xyz/keyshade/blob/59e428002b3b9bb88908da6ffa81605de7d2b3dd/packages/schema/src/workspace.ts#L3C1-L7C3